models/safe/zkevm/safe_zkevm_schema.yml

```diff
@@ -0,0 +1,143 @@
+version: 2
+
+models:
+  - name: safe_zkevm_safes
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'zkevm']
+    description: "Safe addresses"
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "The blockchain on which the Safe is deployed"
+      - &address
+        name: address
+        description: "Safe contract address"
+      - &creation_version
+        name: creation_version
+        description: "Version of initially created safe"
+      - &block_date
+        name: block_date
+      - &block_month
+        name: block_month
+      - &creation_time
+        name: creation_time
+        description: "Date/time of safe creation"
+      - &tx_hash
+        name: tx_hash
+
+  - name: safe_zkevm_matic_transfers
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'transfers', 'zkevm']
+    description: "MATIC transfers for safes"
+    columns:
+      - *blockchain
+      - &symbol
+        name: symbol
+        description: "Symbol of native gas token: MATIC"
+      - *address
+      - *block_date
+      - *block_month
+      - &block_time
+        name: block_time
+        description: "Date of MATIC transfer"
+      - &amount_raw
+        name: amount_raw
+        description: "Raw amount of transferred MATIC"
+      - *tx_hash
+      - &trace_address
+        name: trace_address
+      - &amount_usd
+        name: amount_usd
+        description: "USD amount of transferred MATIC"
+
+  - name: safe_zkevm_singletons
+    meta:
+      blockchain: zkevm
+      project: safe
+      contributors: danielpartida
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
+    config:
+      tags: ['safe', 'singletons', 'zkevm']
+    description: "Singletons addresses used with Safes"
+    columns:
+      - name: address
+        description: "Safe contract address"
+        tests:
+          - unique
+          - not_null
+
+  - name: safe_zkevm_transactions
```

Comment:
please add unique combo of columns test here on the unique keys

Reply:
Hi @jeff-dude,

I will take a look at your suggestions next week. Regarding unique combo of columns test, I am not sure what is meant there. 

Also, I would like to understand which parts of this PR comments are necessary to change and which "nice to have" for the future

Reply:
tests like [this](https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/_schema.yml#L11-L18), which runs a query to validate unique rows based on the columns used in your model `unique_keys` property in config block.

Reply:
i consider these changes a must prior to merge. we should avoid as much tech debt as we can, as spellbook has built up quite a bit over time already. it would be ideal to refactor most safe spells to latest patterns, but not required for this PR

Reply:
majority of the latest patterns in spellbook are documented [here](https://github.com/duneanalytics/spellbook/tree/main/docs)

Reply:
Okay, we are super super busy at the moment next week with our listing. Is there a Dune wizard that could help on this end? We would benefit a lot from understanding the suggested changes and take them into consideration for future spells

Reply:
we don't have access to your branch. if we get access, maybe @Hosuke can help? otherwise, i may be able to some point next week

Reply:
Yes, I was not able to directly commit to this branch.
Once I can, I can definitely help here.

Reply:
Please allow maintainers' edits in future PRs.